articles/architecture-scenarios/_includes/_logout/_introduction.md

```diff
@@ -20,6 +23,12 @@ The functionality provided by your application, as well as your use of features
 If the logout feature in one application terminates an Auth0 SSO session that is used by other applications, the user may lose work if they have uncommitted transactions. Be sure to add the functionality needed to handle such conditions to minimize the likelihood of lost work.
 :::
 
+<% if (platform === "b2b") { %>
+In some situations, a user may expect to be logged out of all of their applications when they log out of a single application in your system.  This is something that can add complexity to a system, but if you have sensitive data and are concerned that a user may leave themselves open after logging out because they didn't realize they had to log out of all of their applications, you will need to review [Single Logout](#single-logout).
```

Comment:
```suggestion
In some situations, a user may be expected to logout of all associated applications when they log out of any one of the applications you provide. This is something that can add complexity to a system. However, if you have concerns users could leave themselves vulnerable - perhaps due to data sensitivity, or the like - because they didn't explicitly logout of all associated applications, then you will need to review [Single Logout](#single-logout) and implement accordingly.
```